portal-api/src/scheduler/jobs/send-reporting-period-start-emails-job/index.test.js

```diff
@@ -0,0 +1,96 @@
+const { produce } = require('immer');
+const sendReportingPeriodStartEmailsJob = require('./index');
+const api = require('../../../v1/api');
+const externalApi = require('../../../external-api/bank-holidays');
+const sendEmail = require('../../../external-api/send-email');
+const BANKS = require('../../../../test-helpers/mock-banks');
+const { EMAIL_TEMPLATE_IDS } = require('../../../constants');
+
+jest.mock('../../../v1/api');
+jest.mock('../../../external-api/bank-holidays');
+jest.mock('../../../external-api/send-email', () => jest.fn());
+
+console.error = jest.fn();
+console.warn = jest.fn();
+console.info = jest.fn();
+
+const sendReportingPeriodStartEmailsTask = sendReportingPeriodStartEmailsJob.init().task;
+
+describe('sendReportingPeriodStartEmailsJob', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('does not send an email when the bank has no payment officer team email', async () => {
+    // Arrange
+    const bankWithoutPaymentOfficerTeam = produce(BANKS.HSBC, (draftBank) => {
+      delete draftBank.paymentOfficerTeam.email;
+    });
+
+    api.getAllBanks.mockResolvedValue([bankWithoutPaymentOfficerTeam]);
+    externalApi.getBankHolidaysForRegion.mockResolvedValue([]);
+
+    // Act
+    await sendReportingPeriodStartEmailsTask();
+
+    // Assert
+    expect(sendEmail).not.toHaveBeenCalled();
+    expect(console.warn.mock.calls).toContainEqual([expect.stringContaining('no payment officer team email set')]);
+  });
+
+  it('does not send an email when the bank has and invalid payment officer team email', async () => {
```

Comment:
Just a small typo here
```suggestion
  it('does not send an email when the bank has an invalid payment officer team email', async () => {
```